### ISSUE TITLE
Update the skill roller user interface to allow dice purchase / luck spend before rolling, etc.

### ISSUE THREAD

**Issue by Muttley** [Muttley/foundryvtt-fallout#39]
A similar kind of idea to the Conan dice roller really.

**Comment by bb46003** [comment]
I will handel this

**Comment by Muttley** [comment]
> I will handel this

Great stuff.  Have a look at the Skill Roller in the Conan system, that's the kind of thing I'm aiming for.

**Comment by guver** [comment]
This is resolved right?

**Comment by Muttley** [comment]
> This is resolved right?

No, it's something that is planned to be done at some point in the future.
